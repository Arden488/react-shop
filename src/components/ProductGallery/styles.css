.product-gallery {
  width: 500px;
  height: 500px;
  position: relative;
}

.product-gallery__item {
  display: none;
  position: absolute;
  top: 0;
  left: 0;
}

.product-gallery__item_active {
  display: block;
}
