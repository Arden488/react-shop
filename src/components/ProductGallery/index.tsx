import * as React from 'react';

import './styles.css';

const ProductGallery = () => (
  <div className="product-gallery">
    <img src="/img/iphone1.webp" alt="" className="product-gallery__item product-gallery__item_active" />
    <img src="/img/iphone2.webp" alt="" className="product-gallery__item" />
    <img src="/img/iphone3.webp" alt="" className="product-gallery__item" />
    <img src="/img/iphone4.webp" alt="" className="product-gallery__item" />
  </div>
);

export default ProductGallery;
