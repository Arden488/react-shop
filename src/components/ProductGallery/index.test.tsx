import { shallow } from 'enzyme';
import * as React from 'react';

import ProductGallery from './index';

let component: React.ReactNode = null;

beforeEach(() => {
  component = shallow(<ProductGallery />);
});

it('renders correctly', () => {
  expect(component).toMatchSnapshot();
});
