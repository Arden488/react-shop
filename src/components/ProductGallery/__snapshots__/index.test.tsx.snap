// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="product-gallery"
>
  <img
    alt=""
    className="product-gallery__item product-gallery__item_active"
    src="/img/iphone1.webp"
  />
  <img
    alt=""
    className="product-gallery__item"
    src="/img/iphone2.webp"
  />
  <img
    alt=""
    className="product-gallery__item"
    src="/img/iphone3.webp"
  />
  <img
    alt=""
    className="product-gallery__item"
    src="/img/iphone4.webp"
  />
</div>
`;
